adserver/admin.py

```diff
@@ -662,6 +664,10 @@ def action_create_draft_invoice(self, request, queryset):
             customer=advertiser.djstripe_customer.id,
             auto_advance=False,  # Draft invoice
             collection_method="send_invoice",
+            # Check just under 3k just in case there's a rounding issue
+            description="Includes 10% volume discount"
+            if total_cost >= 290_000
+            else None,
```

Comment:
```suggestion
            description="Includes 10% volume discount"
            if total_cost >= 290_000
            else "Thanks for your business!",
```

This syntax was a bit wonky when I first read it, but I get it now :)